index.html

```diff
@@ -87,10 +87,18 @@ <h1 class="text-muted" id="navbar-title"></h1>
 
     <div class="row marketing">
       <div class="col-lg-6">
-        <h4><a href="https://github.com/Open-MBEE/mdk">Model Development Kit</a></h4>
-        <p>MDK is a plugin for <a target="_blank" href="https://www.nomagic.com/products/magicdraw">MagicDraw</a> that's
-          primary purposes are to sync SysML models with the MMS and implement the DocGen language, which allows
-          modelers to dynamically generate documents in a model-based approach using the view and viewpoint concept.</p>
+        <h4>Model Development Kit</h4>
+        <p>MDKs are tool-specific integrations that's
+          primary purposes are to sync models with the MMS and implement the DocGen language, which allows
+          modelers to dynamically generate documents in a model-based approach using the view and viewpoint concept.
+          <ul>
+          <li><a href="https://github.com/Open-MBEE/mdk">Cameo Systems Modeler MDK</a></li>
+          <li><a href="https://github.com/Open-MBEE/jupyter-mdk">Jupyter MDK</a></li>
+          <li><a href="https://github.com/Open-MBEE/mms-mathematica-client">Mathematica MDK</a>,
+          <li><a href="https://github.com/Open-MBEE/mathematica-mdk-core">Mathematica MDK Core</a></li>
```

Comment:
github description for this says it's deprecated? could be it's been replaced by just the mathematica mdk